back/app/jobs/log_activity_job.rb

```diff
@@ -6,6 +6,24 @@ class LogActivityJob < ApplicationJob
 
   attr_reader :item, :action, :user, :acted_at, :options, :activity
 
+  rescue_from(ActiveJob::DeserializationError) do |exception|
```

Comment:
Does it mean that if the user was deleted, the activity won't be logged? It seems we don't use any user fields in this job. Can we pass the user ID instead of the user object? This way, it doesn't need to be loaded from the DB and so it won't raise an exception.

Reply:
I would expect not, but now I see that we have:
```
LogActivityJob.perform_later(encode_frozen_resource(frozen_user), 'deleted', current_user, Time.now.to_i)
```

So the item itself is fine, because it gets serialized as a parameter (so it doesn't need to exist when the job runs). However, if a user deletes their own account, I wonder if the current_user is set to that user. If that's the case it would be good to fix this.